flink-tests/src/test/java/org/apache/flink/test/checkpointing/WindowCheckpointingITCase.java

```diff
@@ -92,19 +89,21 @@ private static Configuration getConfiguration() {
 	@Test
 	public void testTumblingProcessingTimeWindow() {
 		final int numElements = 3000;
-		FailingSource.reset();
 
 		try {
 			StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
 			env.setParallelism(PARALLELISM);
 			env.setStreamTimeCharacteristic(timeCharacteristic);
 			env.getConfig().setAutoWatermarkInterval(10);
 			env.enableCheckpointing(100);
-			env.setRestartStrategy(RestartStrategies.fixedDelayRestart(3, 0));
+			env.setRestartStrategy(RestartStrategies.fixedDelayRestart(1, 0));
```

Comment:
This is reduced from 3 to 1 to reduce runtime of the test?

Reply:
Yip, and it is also no longer required because we do no longer work with `SuccessException` if it is not required.